bip-amkn-posthyb.mediawiki

```diff
@@ -0,0 +1,167 @@
+<pre>
+  BIP: amkn-posthyb
+  Title: Consensus (Hard Fork) PoST Datastore for Advanced Cryptography and Higher Efficiency Mining
+  Author: Andrew M. K. Nassief <loneroassociation@gmail.com>
+  Comments-Summary: General interests and concerns, community has been looking into more details
+  Comments-URI: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2021-March/018571.html
+  Status: Active
+  Type: Draft
+  Created: 2021-03-04
+  License: MIT | zlib | OPNL-2 | CC BY 4.0
+  Replaces: N/A
+</pre>
+
+==Abstract==
+This BIP aims to provide a more complex cryptography alternative to Bitcoin's traditional encryption scheme. It also specifies a hybrid consensus for allowing both traditional PoW and memory hard mining. The aim is for more optimized cryptographic efficiency and sustainable security in the future.
```

Comment:
The bitcoin consensus layer doesn't have any primitives to deal with encryption. Mining involving encryption is a common misconception spread by non-technical journalists that are unable to proof-read their writing. I recommend that you invest time to learn about the basics of bitcoin first. Good resources are the Bitcoin Stack Exchange and the bitcoin-dev mailing list.

Reply:
Hi, did you read the entire BIP?  This proposal changes include both specific updates to the cryptography and specific updates for the mining which are both primarily related to block and transaction validation.

Reply:
Nonetheless, I decided to update the wording slightly.